### PR TITLE
add installation website,add notebook, uninstall torchvision, update torch

### PR DESCRIPTION
**Pull request by StevenJokes**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1304/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

File: chapter_installation/index.md [hunk 1]
```diff
@@ -96,7 +97,8 @@ pip install mxnet==1.6.0
 :begin_tab:`pytorch`
 
 ```bash
-pip install torch==1.5.1 torchvision -f https://download.pytorch.org/whl/torch_stable.html
+# https://pytorch.org/get-started/locally/
+pip install torch==1.6.0+cpu torchvision==0.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (AnirudhDagar) [inline chapter_installation/index.md:101]: Why force a CPU install?

**Comment by StevenJokes** [comment]
I think it would be better to handle cpu and gpu version separately.
We introduce cpu first then gpu.

**Comment by AnirudhDagar** [comment]
Let's just follow the way it is in Tensorflow.

Besides for PyTorch, pip install torch by default installs a GPU version if it's present otherwise a CPU version.

**Comment by StevenJokes** [comment]
You can add your words in editing my PR files.
image

**Comment by StevenJokes** [comment]
@AnirudhDagar @astonzhang 
Better now?

**Comment by mli** [comment]
Job d2l-en/PR-1304/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/10 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/11 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/12 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/13 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by mli** [comment]
Job d2l-en/PR-1304/14 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by StevenJokes** [comment]
@astonzhang Can you take a look?

**Comment by mli** [comment]
Job d2l-en/PR-1304/15 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1304/

**Comment by astonzhang** [comment]
I'm going to make some edits after merging this.

**Pull request merged**